app/src/main/java/com/cornellappdev/volume/ui/components/general/PublicationComponents.kt

```diff
@@ -145,3 +156,53 @@ fun CreateHorizontalPublicationRow(
         }
     }
 }
+
+@Composable
+fun CreateFollowPublicationRow (
+    publication: Publication,
+    onPublicationClick: (Publication) -> Unit
+){
+    val title= publication.name
+    Column(modifier = Modifier
+        .wrapContentHeight()
+        .width(100.dp)
+        .clickable {
+            onPublicationClick(publication)
+        }){
+        AsyncImage(
+            model = publication.profileImageURL, modifier = Modifier
+                .height(100.dp)
+                .width(100.dp)
+                .clip(CircleShape), contentDescription = null, contentScale = ContentScale.Crop
+        )
+        Text(
+            modifier = Modifier.padding(bottom = 2.dp, top = 2.dp),
+            text = title,
+            maxLines = 3,
+            overflow = TextOverflow.Ellipsis,
+            fontFamily = notoserif,
+            fontWeight = FontWeight.Medium,
+            fontSize = 12.sp,
+            textAlign = TextAlign.Center
+        )
+
+    }
+}
+
+@Composable
+fun createMorePublicationsRow(
```

Comment:
nit: Delete this if it isn't in use. Try and only check into the codebase things in use/things not commented out